src/components/Hashcat/Hashcat.tsx

```diff
@@ -200,104 +171,74 @@ const Hashcat = () => {
                 </div>
             )}
 
-            <Stack>
-                {UserGuide(title, description_userguide)}
-                <NativeSelect
-                    value={selectedModeOption}
-                    onChange={(e) => setSelectedModeOption(e.target.value)}
-                    title={"Attack mode"}
-                    data={attackMode}
-                    required
-                    placeholder={"Pick a attack mode"}
-                    label={"Attack mode, refer: https://hashcat.net/hashcat"}
-                />
-                <Grid>
-                    <Grid.Col span={3}>
-                        <NativeSelect
-                            value={selectedInputOption}
-                            onChange={(e) => setSelectedInputOption(e.target.value)}
-                            label={"Input Hash Type"}
-                            data={inputType}
-                            required
-                            placeholder={"Input type"}
-                        />
-                    </Grid.Col>
-                    <Grid.Col span={9}>
-                        <NumberInput
-                            label={"Hash Algorithm, refer: https://hashcat.net/hashcat"}
-                            placeholder={"Enter the Hash Algorithm code"}
-                            required
-                            {...form.getInputProps("hashAlgorithmCode")}
-                        />
-                    </Grid.Col>
-                </Grid>
-                {!isFile && (
-                    <TextInput
-                        label={"Input hash value"}
-                        placeholder={"eg:ae71a4222ef1893879b442f14c40c370"}
+            <Grid>
+                <Grid.Col span={6}>
+                    <NativeSelect
+                        data={attackModeOptions}
+                        label="Attack Mode"
+                        placeholder="Pick one"
+                        value={selectedModeOption}
+                        onChange={(event) => setSelectedModeOption(event.currentTarget.value)}
                         required
-                        {...form.getInputProps("hashValue")}
                     />
-                )}
-                {isFile && (
-                    <TextInput
-                        label={"Input hash file path"}
-                        placeholder={" eg: /root/hash.txt"}
+                </Grid.Col>
+                <Grid.Col span={6}>
+                    <NativeSelect
+                        data={inputTypeOptions}
+                        label="Input Type"
+                        placeholder="Pick one"
+                        value={selectedInputOption}
+                        onChange={(event) => setSelectedInputOption(event.currentTarget.value)}
                         required
-                        {...form.getInputProps("hashFilePath")}
                     />
-                )}
+                </Grid.Col>
+            </Grid>
+
+            <Stack>
```

Comment:
You missed LoadingOverlayAndCancelButtonPkexec
 
```suggestion
 <Stack>
    {LoadingOverlayAndCancelButtonPkexec(loading, pid, handleProcessData, handleProcessTermination)}
```